lib/grandpa/votes_tracker_test.go

```diff
@@ -70,7 +70,9 @@ func Test_newVotesTracker(t *testing.T) {
 	}
 	vt := newVotesTracker(capacity)
 
-	assert.Equal(t, expected, vt)
+	assert.Equal(t, expected.mapping, vt.mapping)
+	assert.Equal(t, expected.linkedList, vt.linkedList)
+	assert.Equal(t, expected.capacity, vt.capacity)
```

Comment:
Revert this once the mutex is a field and not embedded in the tracker.
The point of this assertion is to assert everything returned, asserting field by field opens room for errors for future modifications.

Reply:
resolving embedding related comments